### PR TITLE
Remove items that has never been supported

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #6809 

@vinyldarkscratch Could these be checked via mdn-bcd-collector results? I found at least one item that actually has a support (`AuthenticatorAttestationResponse.prototype.getTransports`) and I'm sure there are several more.

### CODE REVIEW

**Comment by foolip** [comment]
@saschanaz this is a great PR and much needed cruft cleanup, and cross checking against collector results to make sure we're not deleting entries that actually have been implemented is a great idea. It's a pretty big PR to review/land at once though, could it be split into smaller bits? Did you use a script to produce it, so that it could be recreated on master at any time?

@ddbeck thoughts on this, given that you marked it not ready? What are the checks you'd like to see, other than careful manual review of each removal?

**Comment by foolip** [comment]
As a compat overview, here are the entries removed by this PR:

api.AuthenticatorAttestationResponse.getTransports
api.BluetoothAdvertisingData
api.BluetoothDevice.adData
api.BluetoothDevice.connectGATT
api.BluetoothDevice.deviceClass
api.BluetoothDevice.gattServer
api.BluetoothDevice.paired
api.BluetoothDevice.productID
api.BluetoothDevice.productVersion
api.BluetoothDevice.unwatchAdvertisements
api.BluetoothDevice.uuids
api.BluetoothDevice.vendorID
api.BluetoothDevice.vendorIDSource
api.BluetoothDevice.watchAdvertisements
api.BluetoothDevice.watchingAdvertisements
api.BluetoothRemoteGATTService.getCharacteristics
api.BluetoothRemoteGATTService.getIncludedService
api.BluetoothRemoteGATTService.isPrimary
api.BudgetService.getBudget
api.BudgetService.getCost
api.BudgetState
api.CSS.ic
api.CSS.lh
api.CSS.rlh
api.CSS.vb
api.CSS.vi
api.CanvasRenderingContext2D.addHitRegion.other_hit_region_options
api.CanvasRenderingContext2D.currentTransform.DOMMatrix_return_value
api.ConstrainULong
api.DOMConfiguration
api.DOMImplementationList
api.Document.dragexit_event
api.Document.mozSyntheticDocument
api.DocumentFragment.methods
api.DocumentOrShadowRoot.nodeFromPoint
api.DocumentOrShadowRoot.nodesFromPoint
api.HTMLButtonElement.menu
api.HTMLIFrameElement.setNfcFocus
api.HTMLIsIndexElement
api.HTMLMenuItemElement.command
api.HTMLTableElement.sortable
api.HTMLTableElement.stopSorting
api.IDBVersionChangeRequest.setVersion
api.NDEFMessage
api.NDEFReader
api.NDEFReadingEvent
api.NDEFRecord
api.NDEFWriter
api.PaymentAddress.regionCode
api.PerformanceFrameTiming
api.RTCIceTransport.gatheringstatechange_event
api.RTCIceTransport.getLocalCandidates
api.RTCIceTransport.getLocalParameters
api.RTCIceTransport.ongatheringstatechange
api.RTCIceTransport.onselectedcandidatepairchange
api.RTCIceTransport.onstatechange
api.RTCIceTransport.selectedcandidatepairchange_event
api.RTCIceTransport.statechange_event
api.RTCRemoteOutboundRtpStreamStats.reportsSent
api.RTCStatsReport
api.Range.compareNode
api.ReadableByteStreamController
api.ReadableStreamBYOBReader
api.ReadableStreamBYOBRequest
api.SVGMeshElement
api.ULongRange
api.Window.clipboardchange_event
api.Window.getAttention
api.Window.getAttentionWithCycleCount
api.Window.mozAnimationStartTime
api.Window.vrdisplayblur_event
api.XRPermissionDescriptor
api.XRPermissionStatus
css.at-rules.media.prefers-reduced-transparency
css.at-rules.media.scan
css.at-rules.media.scripting
css.at-rules.media.update
css.at-rules.page.bleed
css.at-rules.page.marks
css.at-rules.page.page-margin-boxes
css.at-rules.page.size
css.at-rules.viewport.viewport-fit
css.properties.azimuth
css.properties.break-after.multicol_context.always
css.properties.break-after.paged_context.recto
css.properties.break-after.region_context
css.properties.break-before.multicol_context.always
css.properties.break-before.paged_context.recto
css.properties.break-before.region_context
css.properties.break-inside.region_context
css.properties.column-fill.balance-all
css.properties.column-width.intrinsic_sizes
css.properties.font-variant.greek_accented_characters
css.properties.font-variant.uppercase_eszett
css.properties.initial-letter-align
css.properties.margin-trim
css.properties.mask-origin.fill-box
css.properties.mask-origin.stroke-box
css.properties.mask-origin.view-box
css.properties.offset-position
css.properties.ruby-position.inter-character
css.properties.text-align.justify-all
css.properties.text-align.string
css.properties.text-indent.each-line
css.properties.text-indent.hanging
css.properties.text-overflow.fade_function
css.properties.text-overflow.fade_value
css.selectors.blank
css.selectors.column
css.selectors.empty.matches_whitespace
css.selectors.first-letter.dutch_ij_digraph
css.selectors.grammar-error
css.selectors.has
css.selectors.spelling-error
css.types.attr.fallback
css.types.attr.type-or-unit
css.types.frequency-percentage
css.types.frequency
css.types.image.image
css.types.length-percentage.cap
css.types.length-percentage.ic
css.types.length-percentage.lh
css.types.length-percentage.rlh
css.types.length-percentage.vb
css.types.length-percentage.vi
css.types.length.cap
css.types.length.ic
css.types.length.lh
css.types.length.rlh
css.types.length.vb
css.types.length.vi
html.elements.command
html.elements.dir
html.elements.iframe.allowpaymentrequest
html.elements.isindex
html.elements.link.sizes
html.elements.listing
html.elements.menu.button_menus
html.elements.menu.type.type_toolbar
html.elements.multicol
html.elements.nextid
html.elements.textarea.autocapitalize
http.headers.Cache-Control.stale-if-error
http.headers.csp.Content-Security-Policy.navigate-to
http.headers.csp.Content-Security-Policy.prefetch-src
http.headers.Cookie2
http.headers.Feature-Policy.battery
http.headers.Feature-Policy.layout-animations
http.headers.Feature-Policy.picture-in-picture
http.headers.Feature-Policy.vibrate
http.headers.Feature-Policy.wake-lock
http.headers.Set-Cookie2
javascript.builtins.AsyncIterator.AsyncIterator
javascript.builtins.AsyncIterator.asIndexedPairs
javascript.builtins.AsyncIterator.drop
javascript.builtins.AsyncIterator.every
javascript.builtins.AsyncIterator.filter
javascript.builtins.AsyncIterator.find
javascript.builtins.AsyncIterator.flatMap
javascript.builtins.AsyncIterator.forEach
javascript.builtins.AsyncIterator.from
javascript.builtins.AsyncIterator.map
javascript.builtins.AsyncIterator.reduce
javascript.builtins.AsyncIterator.some
javascript.builtins.AsyncIterator.take
javascript.builtins.AsyncIterator.toArray
javascript.builtins.AsyncIterator.@@toStringTag
javascript.builtins.Iterator.Iterator
javascript.builtins.Iterator.asIndexedPairs
javascript.builtins.Iterator.drop
javascript.builtins.Iterator.every
javascript.builtins.Iterator.filter
javascript.builtins.Iterator.find
javascript.builtins.Iterator.flatMap
javascript.builtins.Iterator.forEach
javascript.builtins.Iterator.from
javascript.builtins.Iterator.map
javascript.builtins.Iterator.reduce
javascript.builtins.Iterator.some
javascript.builtins.Iterator.take
javascript.builtins.Iterator.toArray
javascript.builtins.Iterator.@@toStringTag
javascript.operators.pipeline
mathml.elements.math.overflow
mathml.elements.mglyph
mathml.elements.mlabeledtr
mathml.elements.mo.form
mathml.elements.mo.largeop
mathml.elements.mspace.linebreak
mathml.elements.mstyle.decimalpoint
mathml.elements.mstyle.infixbreakstyle
mathml.elements.mtable.alignmentscope
mathml.elements.mtable.columnwidth
mathml.elements.mtable.equalcolumns
mathml.elements.mtable.equalrows
mathml.elements.mtable.groupalign
mathml.elements.mtable.minlabelspacing
mathml.elements.mtable.side
mathml.elements.mtd.groupalign
mathml.elements.mtr.groupalign
svg.elements.textPath.method
webextensions.manifest.content_security_policy.isolated_world
webextensions.manifest.content_security_policy.sandbox
webextensions.manifest.manifest_version.v3
webextensions.match_patterns.scheme.ftps

**Comment by saschanaz** [comment]
> Did you use a script to produce it, so that it could be recreated on master at any time?

Yes, this is from a customized version of #6809 where things are autodeleted. I'll try splitting this into smaller pieces.

**Comment by ddbeck** [comment]
Sorry, I apologize for forgetting to say why I marked it as not ready. I was going to suggest that we put it into draft status, like a few other PRs, and use it as a burndown for removing these topic-by-topic. For all-`false` features, I'm just not confident we can mass delete them. I think we need to do a little bit of work to make sure that there really was no implementation.

Alternatively, if we want to qualify each of these features within a big PR like this, I'm thinking we ought to hold back on this until the `@mdn` migration is done and then treat this as a semver minor version event.

**Comment by ddbeck** [comment]
Oh, also, there's a bit of work to do on MDN: if there exist actual pages for any of these, we should archive them. That's another thing I wanted to check before merging any of this.

**Comment by foolip** [comment]
I think review will inevitably turn up some issues that will take some back and forth, so splitting into a few groups would be great. I'd be happy to review smaller PRs and do some source code archeology for Blink, Gecko and WebKit to ensure no implementation has existed.

**Comment by saschanaz** [comment]
Thanks for the feedback. First things first, converting this to draft:

**Comment by foolip** [comment]
https://github.com/mdn/browser-compat-data/pull/6929 has a discussion of how we're going to deal with these PRs. Summarizing it here:
- Someone reviews to confirm historical non-implementation of the removed entries
- That someone approves and pings a second reviewer, asking them to:
  - Redirect the page to the parent interface where possible (@ddbeck can do this)
  - Otherwise archive or delete on a case-by-case basis, leaning towards deletion unless there's clear value on the page (@chrisdavidmills can do this)
- That second reviewer does the redirect/archive/delete and then merges the PR

**Comment by ddbeck** [comment]
One additional note: we need the [needs-release-note :newspaper:](https://github.com/mdn/browser-compat-data/labels/needs-release-note%20%3Anewspaper%3A) label on the PRs with removals.

**Comment by foolip** [comment]
@saschanaz does this still include any removals that haven't been split out to separate PRs?

**Comment by saschanaz** [comment]
Probably not, but I'll check when everything is merged.

**Comment by Mouvedia** [comment]
`lh` and `rlh` units are partially supported by Safari.

https://webkit.org/blog/10428/release-notes-for-safari-technology-preview-105/
https://bugs.webkit.org/show_bug.cgi?id=211351

**Comment by foolip** [comment]
@saschanaz I'd like to review some of these changes for actually merging them. Would you like to split this into smaller pieces, or should I comment on bits I can confirm and then you can split out just those?

**Comment by saschanaz** [comment]
Some of them are already splitted as #6952 #6955 #6956 #6957, others are still to be covered. Will try...

**Comment by foolip** [comment]
Ah, I was mainly interested in the API data, perhaps the only part that hasn't been split out :)

@Elchi3 has also sent PRs for some HTML stuff that probably overlaps, but that's OK since conflicts aren't much of a problem.

**Comment by saschanaz** [comment]
14 changed files in `api/`, shouldn't be hard to split them. Doing it right now:

File: api/SourceBuffer.json [hunk 1]
```diff
@@ -1111,54 +1111,6 @@
           }
         }
       },
-      "trackDefaults": {
```
Comment (foolip) [inline api/SourceBuffer.json:1114]: I extracted this removal into https://github.com/mdn/browser-compat-data/pull/12703

File: api/Window.json [hunk 2]
```diff
@@ -1694,54 +1694,6 @@
           }
         }
       },
-      "dialogArguments": {
```
Comment (foolip) [inline api/Window.json:1697]: I'd be happy to approve this removal if put in a separate PR.

File: api/Window.json [hunk 3]
```diff
@@ -8572,54 +8524,6 @@
           }
         }
       },
-      "setCursor": {
```
Comment (foolip) [inline api/Window.json:8575]: I'd be happy to approve this removal if put in a separate PR.

File: api/Window.json (hunk 2 above)
Comment (saschanaz) [reply api/Window.json:1697]: #12704

File: api/VRDisplay.json [hunk 4]
```diff
@@ -1051,54 +1051,6 @@
           }
         }
       },
-      "hardwareUnitId": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/VRDisplay/hardwareUnitId",
```
Comment (saschanaz) [inline api/VRDisplay.json:1056]: I doubt anyone is willing to add anything for deprecated WebVR, what do you think?

Comment (foolip) [reply api/VRDisplay.json:1056]: I tried to remove this entry but got stuck cleaning up MDN. It's referenced in examples that I didn't know how to rewrite.

**Comment by XSpielinbox** [comment]
> As a compat overview, here are the entries removed by this PR:
> 
> http.headers.csp.Content-Security-Policy.navigate-to
http.headers.csp.Content-Security-Policy.prefetch-src
http.headers.Cookie2 http.headers.Feature-Policy.battery
http.headers.Feature-Policy.layout-animations
http.headers.Feature-Policy.picture-in-picture
http.headers.Feature-Policy.vibrate
http.headers.Feature-Policy.wake-lock

1. I cannot find any data at the moment for the `navigate-to`, `layout-animations` or `vibrate` directives.
2. `prefetch-src`, `battery`, `picture-in-picture` and `screen-wake-lock` directives seem to be supported at least by some browsers (though as Permissions-Policy). I would therefore not remove them, but rather Firefox should implement support for them too.
